01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
 <!-- http://www.nytimes.com/
 Deceptively complex layout. This will be good practice for you to get experience with HMTL5 semantic elements.
--->
\ No newline at end of file
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>The New York Times - Breaking News, World News, & Multimedia</title>
+</head>
+
+<body>
+  <header>
+    <nav id="side-full-menu" class="inline-block">
+      <div>
+        <img href="http://marketblog.envato.com/wp-content/uploads/2014/11/resp_nav_a.jpg" height="30" alt="hamburger-menu">SECTIONS</div>
+      <ul class="no-text-decoration onmouseover">
+        <li><a href="http://www.nytimes.com/">Home Page</a></li>
+        <li><a href="http://www.nytimes.com/pages/world/index.html">World</a></li>
+        <ul>
```

Comment:
Confusingly enough, this `ul` tag should be wrapped inside an `li`. As a rule, a `ul` should never have a direct child that's not an `li`.